src/main/java/edu/whimc/observations/utils/sql/Queryer.java

```diff
@@ -132,7 +133,14 @@ private PreparedStatement getStatement(Connection connection, Observation obs) t
         statement.setBoolean(11, true);
         statement.setLong(12, obs.getExpiration().getTime());
         statement.setString(13, category);
-
+        String obsNoColorWithAmpersand = ChatColor.stripColor(obs.getObservation());
+        String[] splitObsNoColor = obsNoColorWithAmpersand.split("&");
+        String obsNoColor = splitObsNoColor[0];
+        for(int k = 1; k < splitObsNoColor.length; k++)
+        {
+            obsNoColor += splitObsNoColor[k].substring(1);
+        }
+        statement.setString(14, obsNoColor);
```

Comment:
```suggestion
        statement.setString(14, ChatColor.stripColor(Utils.color(obs.getObservation())));
```
A cleaner way to do this would be to color the string then strip the colors so you don't have to manually do any string parsing.